routes/topic-router.js

```diff
@@ -0,0 +1,12 @@
+const topicRouter = require('express').Router(); 
+const { fetchTopics } = require('../models/topic-models.js');
+
+topicRouter
+    .route('/')
+    .get((req, res) => {
+        return fetchTopics().then((topics) => {
+            res.status(200).send(topics); 
+        })
+    }); 
```

Comment:
Routers are great but it's important to have your controllers in a separate directory for separation of concerns. Please can you extract this :)